lacci/lib/shoes/app.rb

```diff
@@ -60,6 +60,18 @@ def initialize(
         end
       end
 
+      if ENV["SHOES_SPEC_TEST"]
+        require "scarpe/components/minitest_export_reporter"
+        Minitest::Reporters::ShoesExportReporter.activate!
+        test_code = File.read ENV["SHOES_SPEC_TEST"]
+        if test_code != ""
```

Comment:
```suggestion
        unless test_code.empty?
```

"do this, unless that file is empty" what do you think